phlib/theme.c

```diff
@@ -2487,6 +2611,46 @@ VOID ThemeWindowRenderTabControl(
             }
         }
     }
+
+    {
+        TabCtrl_GetItemRect(WindowHandle, currentSelection, &itemRect);
+
+        PhOffsetRect(&itemRect, 2, 2);
+        itemRect.bottom += itemRect.bottom + 1 < headerBottom ? 1 : -1;
+        itemRect.right += itemRect.right + 1 < clientRect->right;
+        PhInflateRect(&itemRect, 1, 1);     // draw selected tab slightly bigger
+        itemRect.bottom -= 1;
+        SetDCBrushColor(bufferDc, PhPtInRect(&itemRect, Context->CursorPos) ? PhThemeWindowHighlightColor : RGB(0x50, 0x50, 0x50));
+        FillRect(bufferDc, &itemRect, PhGetStockBrush(DC_BRUSH));
+
+        if (TabCtrl_GetItem(WindowHandle, currentSelection, &tabItem))
+        {
+            DrawText(
+                bufferDc,
+                tabItem.pszText,
+                (UINT)PhCountStringZ(tabItem.pszText),
+                &itemRect,
+                DT_CENTER | DT_VCENTER | DT_SINGLELINE | DT_HIDEPREFIX
+            );
```

Comment:
nit: number of places where `);` is not aligned property for a function call - again don't worry about it unless you end up having to make changes - can be cleaned up after merge by @dmex or I